### PR TITLE
feature/enhanced state survivor round7

### PR DESCRIPTION
**Pull request by ootakazuhiko**
- **test: guard enhanced state import edge cases**
- **docs,test: extend enhanced state import coverage**
- **test: enforce imported version upper bounds**
- **test: tighten enhanced-state-manager import fixtures**
- **test: harden enhanced-state import paths**
- **test: enforce metadata retention on importState**

### CODE REVIEW

File: tests/property/token-optimizer.trim-edge.trailing-comma.boundary.pbt.test.ts [hunk 1]
```diff
@@ -8,14 +8,18 @@ describe('PBT: TokenOptimizer trim-edge trailing comma boundary', () => {
...
       await fc.assert(
-        fc.asyncProperty(fc.string({ minLength: 1, maxLength: 60 }), async (s) => {
-          const opt = new TokenOptimizer();
-          const docs = { product: `${s},  ` } as Record<string, string>;
-          const { compressed } = await opt.compressSteeringDocuments(docs, { maxTokens: 2000 });
-          const last = compressed.trimEnd().slice(-1);
-          expect([',', ';']).not.toContain(last);
-        }),
-        { numRuns: 12 }
+        // 固定シードでStrykerサンドボックスの再現性を確保する
```
[the remark this answers is not in the document]
Comment (ootakazuhiko) [reply tests/property/token-optimizer.trim-edge.trailing-comma.boundary.pbt.test.ts]: Updated the comment to English as suggested.
